src/containers/ConcentricCircles/PromptObstacle.js

```diff
@@ -0,0 +1,38 @@
+import React, { PureComponent } from 'react';
+import { withHandlers, compose } from 'recompose';
+import PropTypes from 'prop-types';
+
+import { PromptSwiper } from '../../containers';
+import { withBounds } from '../../behaviours';
+import { DropObstacle } from '../../behaviours/DragAndDrop';
+
+class PromptObstacle extends PureComponent {
+  render() {
+    const {
+      className,
+      ...rest
+    } = this.props;
+
+    return (
+      <div className={className}>
+        <PromptSwiper {...rest} />
+      </div>
+    );
+  }
+}
+
+PromptObstacle.propTypes = {
+  className: PropTypes.string,
+};
+
+PromptObstacle.defaultProps = {
+  className: '',
+};
+
+export default compose(
+  withBounds,
```

Comment:
Why is `withBounds` needed on these components? I don't see what's relying on the state it provides, but it seems very similar to the update logic in DropObstacle and DropTarget.

Reply:
At one point I thought I needed it, but I don't think we actually do. I'll take that out. Thanks.